src/amo/components/HeroRecommendation/index.js

```diff
@@ -118,24 +120,31 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
           'HeroRecommendation--no-image': !featuredImage,
         })}
       >
-        {featuredImage && (
-          <div>
-            <img
-              className="HeroRecommendation-image"
-              alt=""
-              src={featuredImage}
+        <div className="App-banner HeroRecommendation-banner">
+          <SiteNotices />
+          <SurveyNotice />
```

Comment:
Note: We need to pass `location` into `SurveyNotice`, but I wanted to put together a quick demo of what I got working, so that's something I'll do if we decide this is a good approach.

If we do decide to go in this direction, a new component, called `AppBanner` or something, could be created which wraps both `SiteNotices` and `SurveyNotice`, as they are both included here and in `App`.